packages/back-end/src/integrations/Redshift.ts

```diff
@@ -31,4 +40,54 @@ export default class Redshift extends SqlIntegration {
   ensureFloat(col: string): string {
     return `${col}::float`;
   }
+  async getInformationSchema(): Promise<InformationSchema[]> {
+    const sql = `SELECT
+        table_name,
+        table_catalog,
+        table_schema,
+        count(column_name) as column_count
+      FROM
+        SVV_COLUMNS
+      WHERE
+        table_schema
+      NOT IN ('information_schema')
+      GROUP BY (table_name, table_schema, table_catalog)`;
+
+    const results = await this.runQuery(sql);
+
+    if (!results.length) {
+      throw new Error(`No tables found.`);
+    }
+
+    return formatInformationSchema(
+      results as RawInformationSchema[],
+      "postgres"
```

Comment:
Should this be `redshift`?  Redshift is based on Postgres, so if this was intentional, it's worth adding a comment explaining why this doesn't match the data source type.